browser-test/bin/wait_for_server_start_and_run_tests.sh

```diff
@@ -11,6 +11,10 @@ SERVER_URL="http://civiform:9000"
 # https://playwright.dev/docs/browsers#managing-browser-binaries-1
 PLAYWRIGHT_BROWSERS_PATH=0
```

Comment:
That's no longer needed if you set it inside image? And this script is run only from within container.

Reply:
removed